src/wp-includes/block-patterns.php

```diff
@@ -48,12 +48,17 @@ function _register_core_block_patterns_and_categories() {
  * Register Core's official patterns from wordpress.org/patterns.
  *
  * @since 5.8.0
+ * @since 5.9.0 The $current_screen argument was removed.
  *
- * @param WP_Screen $current_screen The screen that the current request was triggered from.
+ * @param WP_Screen $deprecated Unused. Formerly the screen that the current request was triggered from.
  */
-function _load_remote_block_patterns( $current_screen ) {
-	if ( ! $current_screen->is_block_editor ) {
-		return;
+function _load_remote_block_patterns( $deprecated = null ) {
+	if ( ! empty( $deprecated ) ) {
+		_deprecated_argument( __FUNCTION__, '5.9.0' );
+		$current_screen = $deprecated;
+		if ( ! $current_screen->is_block_editor ) {
+			return;
+		}
 	}
```

Comment:
- Changes here are consistent with other function signature deprecations 👍 
- Retains the original behavior from 5.8 for back-compat 👍